src/background.js

```diff
@@ -78,3 +85,109 @@ if (isDevelopment) {
     })
   }
 }
+
+function playDiscordSound (connection, filePath, volume, offset) {
+  const path = filePath
+  const option = { volume: toRealVolume(volume) }
+  if (offset !== 0) {
+    option['seek'] = offset
+  }
+
+  const broadcast = discordClient.voice.createBroadcast()
+  const dispatcher = broadcast.play(path, option)
+  // const playSound = () => {
+  connection.play(broadcast)
+  dispatcher.on('error', error => { console.error(error) })
+  connection.on('error', error => { console.error(error) })
+  dispatcher.on('finish', () => {
+    playDiscordSound(connection, filePath, toRealVolume(dispatcher.volume), offset)
+  })
+}
+
+ipcMain.on('discordJoin', (event, data) => {
+  discordClient.on('message', async message => {
+    if (message.content === ':soc: join') {
+      if (message.member.voice.channel) {
+        await message.member.voice.channel.join()
+      } else {
+        message.reply('You need to join a voice channel first!')
+      }
+    }
+  })
+
+  const userHome = process.env[process.platform == 'win32' ? 'USERPROFILE' : 'HOME']
+  const pathConfig = path.join(userHome, 'discord_token.ini')
+
+  try {
+    fs.statSync(pathConfig)
+  } catch (error) {
+    if (error.code === 'ENOENT') {
+      const options = {
+	type: 'question',
+	title: 'No Token File',
+	message: 'No Token Config File',
+	detail: `There is no discord token configuration file in the ${pathConfig}.\nDo you want to create a template file?`,
+	buttons: ['OK', 'Cancel'],
+	cancelId: -1
+      }
+      const selected = dialog.showMessageBox(options)
+      if (selected == 0) {
+	fs.writeFile(pathConfig, 'discordToken = "your token"', function (err) {
+	  if (err) throw err
+	  dialog.showMessageBox({type: 'info', detail: `Saved!\nWrite your discord token to ${pathConfig}`})
+	})
+	return
+      } else {
+	return
+      }
+    } else {
+      dialog.showMessageBox({type: 'error', detail: error})
+    }
+  }
+
+  try {
+    var config = ini.parse(fs.readFileSync(pathConfig, 'utf-8'))
```

Comment:
```suggestion
    const config = ini.parse(fs.readFileSync(pathConfig, 'utf-8'))
```

- varが必要になるケースは限りなくゼロに近いです
- letも極めて稀です
- 基本的にconstを使いましょう